qa-heal.planx-pla.net/metadata/aggregate_config.json

```diff
@@ -1113,6 +1113,135 @@
                     "dataset_3_title": "commercial-insurance-and-medicaid-coverage-of-moud-protocol-final.pdf",
                     "dataset_2_title": "moud-coverage-final.xlsx",
                     "dataset_4_title": "summary-report.pdf"
+                },
+                "6169fa0ef5f53c24528b457a": {
+                    "__manifest": [
+                        {
+                            "md5sum": "ac59939d223337502a6463dc6e19e4df",
+                            "file_size": 162135,
+                            "file_name": "covid-19-moud-at-state-correctional-facilities-codebook-021022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/a0a8785a-8663-47b9-95ea-a1813612a2f1"
+                        },
+                        {
+                            "md5sum": "d30e39b94ddf6da73b714afacfd58813",
+                            "file_size": 25499,
+                            "file_name": "covid-19-moud-at-state-correctional-facilities-data-020222.xlsx",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/abe9cd49-fc86-4c9b-b9d0-f8c0280d8aaa"
+                        },
+                        {
+                            "md5sum": "7b9aa1a21f0f23502d104aae17a66c1f",
+                            "file_size": 129324,
+                            "file_name": "covid-19-moud-at-state-correctional-facilities-report-02102022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/e21ea6a6-edf9-467d-b9db-e2c8a55b2cf2"
+                        },
+                        {
+                            "md5sum": "098cbe362b2c7052e2e640ace9d78a3f",
+                            "file_size": 241957,
+                            "file_name": "covid-19-moud-at-state-correctional-facilities-research-protocol-02092022.pdf",
+                            "commons_url": "jcoin.datacommons.io",
+                            "object_id": "dg.6VTS/766b189b-ac97-4341-b8fb-252489a168ce"
+                        }
+                    ],
+                    "authz": "/programs/NIDA/projects/PDAPS",
+                    "data_availability": "available",
+                    "study_url": "https://pdaps.org/datasets/covid-19-mitigation-laws-at-state-correctional-facilities-relating-to-opioi",
+                    "dataset_1_type": "codebook",
+                    "dataset_2_type": "data",
```

Comment:
```suggestion
                    "dataset_2_type": "Data",
```

Reply:
@xritter1  thanks Xenia -- ok if I add your changes when I make the new filters in the next commit?